.gitignore

```diff
@@ -46,6 +46,7 @@ npm-debug.log
 # File-system cruft and temporary files
 .DS_Store
 .idea
+.vscode
```

Comment:
The agreement struck — while getting quite heated at each other **at your desk** — was that we'd move these things to the composer install target, and that a core developer should know what they are doing